src/components/Button/Button.module.css

```diff
@@ -11,49 +11,28 @@
  * 4) Adds a gap between children to automatically handle spacing between icons and text
  */
 .button {
+  @mixin eds-theme-typography-button-label;
   display: inline-flex;
   align-items: center;
   justify-content: center;
   margin: 0; /* 3 */
   gap: 0.75rem; /* 4 */
   border-width: var(--eds-theme-border-width);
   border-style: solid;
-  border-color: var(--eds-theme-color-primary-border);
   border-radius: var(--eds-theme-border-radius);
-  color: var(--eds-theme-color-primary-foreground);
-  background-color: transparent;
   cursor: pointer;
   transition: all var(--eds-anim-fade-quick) var(--eds-anim-ease);
 
-  /* TODO: make default color prop in Icon inherit or something? */
-  svg {
-    fill: currentColor;
-  }
```

Comment:
The `Icon` now has a default color of `currentColor`, which devs can override by passing in some other color into `Icon`.